refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-service/src/main/java/de/muenchen/refarch/integration/s3/configuration/SecurityConfiguration.java

```diff
@@ -50,6 +51,7 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                         .permitAll())
                 .authorizeHttpRequests((requests) -> requests.requestMatchers("/**")
                         .authenticated())
+                .csrf(AbstractHttpConfigurer::disable)
```

Comment:
Why must CSRF be disabled here? Isn't this risky?

Reply:
JWT doesn'T need CSRF and if not disabled services can't communicate with each other.
See also https://github.com/it-at-m/digiwf-core/blob/dev/digiwf-libs/digiwf-spring-security/digiwf-spring-security-core/src/main/java/de/muenchen/oss/digiwf/spring/security/SecurityConfiguration.java#L79 for DigiWF

Reply:
Ok thanks for clarification. ;)

Reply:
After looking a bit around, this should be automatically set if providing Bearer token.
I'll try it again and probably remove it. Was maybe my fault.

Reply:
Reverted the CSRF disable.